scripts/configure-cluster/common/setup-operators.sh

```diff
@@ -31,7 +31,43 @@ install_service_binding_operator() {
 EOF
 }
 
-if [ "$KUBERNETES" == "true" ]; then
+install_service_binding_operator_master() {
+  oc create -f - <<EOF
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: CatalogSource
+  metadata:
+    name: service-binding-master
+    namespace: openshift-marketplace
+  spec:
+    displayName: Service Binding Operator build from master
+    image: quay.io/redhat-developer/servicebinding-operator:index
+    priority: 500
+    publisher: Red Hat
+    sourceType: grpc
+    updateStrategy:
+      registryPoll:
+        interval: 10m0s
+EOF
+  oc create -f - <<EOF
+  apiVersion: operators.coreos.com/v1alpha1
+  kind: Subscription
+  metadata:
+    name: service-binding-operator
+    namespace: openshift-operators
+  spec:
+    channel: candidate
+    installPlanApproval: Automatic
+    name: service-binding-operator
+    source: service-binding-master
+    sourceNamespace: openshift-marketplace
+    startingCSV: service-binding-operator.v1.1.1
```

Comment:
IMO, we should skip setting this. My understanding is that if v1.1.2 is released, this directive here would cause OLM to install v1.1.1, which is not what we want.

Reply:
understood I will update it ASAP